src/dodal/devices/xbpm_feedback.py

```diff
@@ -14,7 +14,7 @@ class XBPMFeedback(Device, Triggerable):
     """The XBPM feedback device is an IOC that moves the DCM, HFM and VFM to automatically
     hold the beam into place, as measured by the XBPM sensor."""
 
-    def __init__(self, prefix: str = "", name: str = "xbpm_feedback") -> None:
+    def __init__(self, prefix: str, name: str = "xbpm_feedback") -> None:
         self.pos_ok = epics_signal_r(float, prefix + "-EA-FDBK-01:XBPM2POSITION_OK")
```

Comment:
#1391 for the XBPM signal refactor, as it also changes from using a single signal to a Motor record (which is making a presumption about how the device works on my part)